### ISSUE TITLE
Android Support for JSR 354 API and Reference Implementation.

### ISSUE THREAD

**Issue by msgilligan** [JavaMoney/javamoney-shelter#17]
Since https://github.com/JavaMoney/javamoney-lib/issues/15 has been closed, I'm opening this issue here.

It would be nice to see the JavaMoney Reference Implementation running on Android so that Bitcoin (and Bitcoin 2.0) Android wallets could use JavaMoney components. It might be possible using [RetroLambda](https://github.com/orfjackal/retrolambda) and/or other tools. Note that there is an (empty) repository, https://github.com/JavaMoney/javamoney-api-bp, for an API back port.

**Comment by keilw** [comment]
Yes, the BP could be made available on Android, too (certain SE 7 stuff like "Diamond" I believe works on Android now, too. Otherwise we should try to set up CI instances for that, too. AFAIK Travis-CI (the preferred CI server for JavaMoney after CloudBees started bugging developers with 15-day timeouts) does not support Android, out of the box, but http://www.codenes.com/blog/?p=391 hints, you can get it to build Android with some adjustments (could need another task for this kind of stuff ;-) )

**Comment by msgilligan** [comment]
I've opened up an [enhancement request](https://code.google.com/p/bitcoinj/issues/detail?id=614) to support this in [bitcoinj](https://bitcoinj.github.io)

**Comment by atsticks** [comment]
Hi
​S​
ean
​

​Since I am preparing the final release all artifacts are available as of
now in Bintray/JFrog:
http://oss.jfrog.org/artifactory/list/oss-release-local/


   - javax.money:money-api(-bp):1.0
   - org.javamoney:moneta(-bp):1.0
   - org.javamoney:javamoney-parent:1.0
   - org.javamoney:javamoney-tck:1.0

​The variants with "-bp" added require Java 7 (the "money-api-bp" is
compatible with Java 6).​

Hope that helps ;)

​Best

Anatole

​

Am 08.03.2015 18:49 schrieb "Sean Gilligan" <notifications@github.com>:

**Comment by keilw** [comment]
Seems some banks and especially one of the largest ATM makers NCR could have great use for JavaMoney on Android;-) http://www.businesscloudnews.com/2015/04/16/ncr-offers-cloud-control-for-android-based-atms/?utm_source=rss&utm_medium=rss&utm_campaign=ncr-offers-cloud-control-for-android-based-atms
